modules/lang-painless/src/main/java/org/elasticsearch/painless/api/Augmentation.java

```diff
@@ -708,11 +708,106 @@ public static Matcher matcher(Pattern receiver, int limitFactor, CharSequence in
     /**
      * Convert a {@link TemporalAccessor} into millis since epoch like {@link Instant#toEpochMilli()}.
      */
-    public static long toEpochMilli(TemporalAccessor v) {
-        return v.getLong(ChronoField.INSTANT_SECONDS) * 1_000 + v.get(ChronoField.NANO_OF_SECOND) / 1_000_000;
+    public static long toEpochMilli(TemporalAccessor receiver) {
+        return receiver.getLong(ChronoField.INSTANT_SECONDS) * 1_000 + receiver.get(ChronoField.NANO_OF_SECOND) / 1_000_000;
+    }
+
+    public static long getMillis(TemporalAccessor receiver) {
+        return toEpochMilli(receiver);
     }
 
     public static DayOfWeek getDayOfWeekEnum(ZonedDateTime receiver) {
         return receiver.getDayOfWeek();
     }
+
```

Comment:
I wonder if we should have a separate file or section these Augmentations that's simply unsupported.

Reply:
Seems like a good idea. I would want to do that in as a follow up, though.